### DISCUSSION TITLE
Input Validation Requirements

### DISCUSSION THREAD

**Discussion by Billy4896** [Billy4896/Tafe230123#3] [General]
Hi all,

After beginning developing the currency converter I noticed a lack of requirements regarding input validation in order to handle incorrect inputs. I.E - string inputs where double inputs are needed to determine the currency conversion. I suggest we implement input validation, however I would like to know if you believe it is necessary despite not being a requirement?

Thank you,

Billy

**Comment by jamiebenson1** [comment]
Hey Billy,
i think that would be an "if we have time" implementation rather than a necessary thing. Absolutely add it if you have time to though.
